packages/dotnet/ReadMe/HarJsonTranslationLogics/ResponseProcessor.cs

```diff
@@ -125,29 +141,72 @@ private string GetStatusTextByStatusCode(int statusCode)
-        case 416: return "Requested Range Not Satisfiable";
+        case 416: return "Range Not Satisfiable";
         case 417: return "Expectation Failed";
+        case 418: return "I'm a teapot";
+        case 419: return "Page Expired"; // Unofficial
+        case 420: return "Enhance Your Calm"; // Unofficial
+        case 421: return "Misdirected Request";
         case 422: return "Unprocessable Entity";
         case 423: return "Locked";
         case 424: return "Failed Dependency";
+        case 425: return "Too Early";
+        case 426: return "Upgrade Required";
+        case 428: return "Precondition Required";
+        case 429: return "Too Many Requests";
+        case 430: return "Request Header Fields Too Large"; // Unofficial
+        case 431: return "Request Header Fields Too Large";
+        case 440: return "Login Time-out"; // Unofficial
+        case 444: return "No Response"; // Unofficial
+        case 449: return "Retry With"; // Unofficial
+        case 450: return "Blocked by Windows Parental Controls"; // Unofficial
+        case 451: return "Unavailable For Legal Reasons";
+        case 494: return "Request Header Too Large"; // Unofficial
+        case 495: return "SSL Certificate Error"; // Unofficial
+        case 496: return "SSL Certificate Required"; // Unofficial
+        case 497: return "HTTP Request Sent to HTTPS Port"; // Unofficial
+        case 498: return "Invalid Token"; // Unofficial
+        case 499: return "Client Error"; // "Token Request" on ArcGIS, "Client Closed Request" on nginx
+
         case 500: return "Internal Server Error";
         case 501: return "Not Implemented";
         case 502: return "Bad Gateway";
         case 503: return "Service Unavailable";
         case 504: return "Gateway Timeout";
-        case 505: return "Http Version Not Supported";
+        case 505: return "HTTP Version Not Supported";
+        case 506: return "Variant Also Negotiates";
         case 507: return "Insufficient Storage";
+        case 508: return "Loop Detected";
+        case 509: return "Bandwidth Limit Exceeded";
+        case 510: return "Not Extended";
+        case 511: return "Network Authentication Required";
+        case 520: return "Web Server Returned an Unknown Error"; // Unofficial
+        case 521: return "Web Server Is Down"; // Unofficial
+        case 522: return "Connection Timed Out"; // Unofficial
+        case 523: return "Origin Is Unreachable"; // Unofficial
+        case 524: return "A Timeout Occurred"; // Unofficial
+        case 525: return "SSL Handshake Failed"; // Unofficial
+        case 526: return "Invalid SSL Certificate"; // Unofficial
+        case 527: return "Railgun Error"; // Unofficial
+        case 529: return "Site is Overloaded"; // Unofficial
+        case 530: return "Site is Frozen"; // Unofficial
+        case 598: return "Network Read Timeout Error"; // Unofficial
       }
-      return "";
-    }
-
 
+      return string.Empty;
```

Comment:
For the rule that was warning on using `""` instead of `string.Empty`: https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1122.md